custom/ilsgateway/tasks.py

```diff
@@ -209,10 +210,12 @@ def report_run(domain):
                 transaction.rollback()
             except:
                 pass
-        run.has_error = True
+        has_error = True
         raise
     finally:
         # complete run
+        run = ReportRun.objects.get(pk=run.id)
```

Comment:
Tasks use copy of runner variable so finally we need to refresh original variable.